static/app/views/issueDetails/groupPriority.tsx

```diff
@@ -36,11 +41,20 @@ function GroupPriority({group}: GroupDetailsPriorityProps) {
       api,
       {
         orgId: organization.slug,
-        projectId: group.project.slug,
```

Comment:
`projectId` isn't required and makes this function use the old project issues endpoint